### ISSUE TITLE
`kpt live status --output=json` is not json

### ISSUE THREAD

**Issue by thackem** [GoogleContainerTools/kpt#2411]
### Expected behavior

A structured JSON string in the output

### Actual behavior

Plaintext, json matches string, matches the default o

### Information

Used kpt beta 1. Package is a simple single resource, intentionally broken to give an InProgress error status. The issue is the --output=json flag not working.

### Steps to reproduce the behavior

```
pkg/bucket$ kpt live status
storagebucket.storage.cnrm.cloud.google.com/blueprints-project-bucket is InProgress: Update call failed: error fetching live state: error reading underlying resource: summary: Error when reading or editing Storage Bucket "blueprints-project-bucket": googleapi: Error 403: GCP account does not have storage.buckets.get access to the Google Cloud Storage bucket., forbidden, detail:

pkg/bucket$ kpt live status --output=json
storagebucket.storage.cnrm.cloud.google.com/blueprints-project-bucket is InProgress: Update call failed: error fetching live state: error reading underlying resource: summary: Error when reading or editing Storage Bucket "blueprints-project-bucket": googleapi: Error 403: GCP account does not have storage.buckets.get access to the Google Cloud Storage bucket., forbidden, detail:

pkg/bucket$ kpt live status --output=table
NAMESPACE   RESOURCE                                  STATUS      CONDITIONS                                AGE     MESSAGE
config-con  StorageBucket/blueprints-project-bucket   InProgress  Ready                                     6m      Update call failed: error fetching live

pkg/bucket$ kpt live status --output=string
storagebucket.storage.cnrm.cloud.google.com/blueprints-project-bucket is InProgress: Update call failed: error fetching live state: error reading underlying resource: summary: Error when reading or editing Storage Bucket "blueprints-project-bucket": googleapi: Error 403: GCP account does not have storage.buckets.get access to the Google Cloud Storage bucket., forbidden, detail:
```

**Comment by thackem** [comment]
actually I think this is fixed in beta 2, I was using beta 1

**Comment by mortent** [comment]
This is a duplicate of https://github.com/GoogleContainerTools/kpt/issues/2269. This issue was fixed in v1.0.0-beta.2.
